prince-of-versions/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version 4.0.3
+
+_2021-04-02_
+
+- fix null value parsing
+    User should receive proper null values when certain fields (ie. version) are null
```

Comment:
Actually, if you think about the change in parsing required and optional versions, user would not see that change I think. That just fixed comparison of the values under the hood.
So, I'd just exclude user from this sentence if possible.